pipenv/envload.py

```diff
@@ -0,0 +1,29 @@
+try:
```

Comment:
does this really need its own file

single function => probably should stay in the main file to avoid the overhead

Reply:
It used to contain a subclass of `StringIO` to abstract away some text stream logic between Python 2 and 3, but I ended up just reading the whole thing out (since the dotenv file would be reasonably small anyway). Also I wanted to trim the 2000-line monster core.py as much as possible… but no, this doesn’t need its own file.